eng/code-quality-reports/src/main/java/com/azure/tools/revapi/transforms/IgnoreCoreAdditionsTransform.java

```diff
@@ -0,0 +1,53 @@
+package com.azure.tools.revapi.transforms;
+
+import org.revapi.Archive;
+import org.revapi.Difference;
+import org.revapi.Element;
+import org.revapi.TransformationResult;
+import org.revapi.base.BaseDifferenceTransform;
+
+import javax.annotation.Nullable;
+import java.util.regex.Pattern;
+
+public class IgnoreCoreAdditionsTransform<E extends Element<E>> extends BaseDifferenceTransform<E> {
```

Comment:
Is there in code of the `IgnoreCoreAdditionsTransform` class say about *core additions*, or it is actually and any change to core?

Ask because 2 rules in exclude appears not *core additions*.
```
java.method.visibilityReduced
java.method.finalMethodAddedToNonFinalClass
```